docs/data_node/extract_node/tube.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TubeMQ
+sidebar_position: 11
+---
+
+import {siteVariables} from '../../version';
+
+## Overview
+
+[Apache InLong TubeMQ](https://inlong.apache.org/docs/modules/tubemq/overview) is a distributed, open source pub-sub messaging and steaming platform for real-time workloads, trillions of massive data precipitation.
+
+## Version
+
+| Extract Node          | Version                                                      |
+| --------------------- | ------------------------------------------------------------ |
+| [TubeMQ](./tube.md) | [TubeMQ](https://inlong.apache.org/docs/next/modules/tubemq/overview): >=0.1.0<br/> |
```

Comment:
Extra `:`

Reply:
Please correct the above format errors  in the Chinese document in the same time.